README.rst

```diff
@@ -36,6 +36,18 @@ To run the tests::
 
     python manage.py test
 
+Translations
+------------
+
+To update translations from Transifex, `pip install transifex-client` and run::
```

Comment:
transifex-client is already in requirements-dev.txt so not sure we need pip install here?